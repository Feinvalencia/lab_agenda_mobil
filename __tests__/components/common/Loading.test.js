import React from 'react';
import renderer from 'react-test-renderer';
import {render, fireEvent} from '@testing-library/react-native';
import Loading from '../../../src/components/common/Loading';

test('Loading snapshot', () => {
  const loading = renderer
    .create(<Loading isVisible={true} text="test 1" />)
    .toJSON();

  expect(loading).toMatchSnapshot();
});
