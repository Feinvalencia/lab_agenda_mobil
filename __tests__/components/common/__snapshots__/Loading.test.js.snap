// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading snapshot 1`] = `
<Modal
  hardwareAccelerated={false}
  onRequestClose={[Function]}
  overlayBackgroundColor="transparent"
  theme={
    Object {
      "colors": Object {
        "disabled": "hsl(208, 8%, 90%)",
        "divider": "#bcbbc1",
        "error": "#ff190c",
        "grey0": "#393e42",
        "grey1": "#43484d",
        "grey2": "#5e6977",
        "grey3": "#86939e",
        "grey4": "#bdc6cf",
        "grey5": "#e1e8ee",
        "greyOutline": "#bbb",
        "platform": Object {
          "android": Object {
            "error": "#f44336",
            "primary": "#2196f3",
            "secondary": "#9C27B0",
            "success": "#4caf50",
            "warning": "#ffeb3b",
          },
          "ios": Object {
            "error": "#ff3b30",
            "primary": "#007aff",
            "secondary": "#5856d6",
            "success": "#4cd964",
            "warning": "#ffcc00",
          },
        },
        "primary": "#2089dc",
        "searchBg": "#303337",
        "secondary": "#8F0CE8",
        "success": "#52c41a",
        "warning": "#faad14",
      },
    }
  }
  transparent={true}
  visible={true}
  windowBackgroundColor="rgba(0, 0, 0, 0.5)"
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(0, 0, 0, .4)",
        "height": "100%",
        "justifyContent": "center",
        "left": 0,
        "position": "absolute",
        "top": 0,
        "width": "100%",
      }
    }
    testID="RNE__Overlay__backdrop"
  />
  <View
    pointerEvents="box-none"
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "#fff",
          "borderColor": "#3368FF",
          "borderRadius": 10,
          "borderWidth": 2,
          "height": 100,
          "padding": 10,
          "shadowColor": "rgba(0, 0, 0, .3)",
          "shadowOffset": Object {
            "height": 1,
            "width": 0,
          },
          "shadowRadius": 4,
          "width": 200,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <ActivityIndicator
          animating={true}
          color="#3368FF"
          hidesWhenStopped={true}
          size="large"
        />
        <Text
          style={
            Object {
              "color": "#3368FF",
              "marginTop": 10,
              "textTransform": "uppercase",
            }
          }
        >
          test 1
        </Text>
      </View>
    </View>
  </View>
</Modal>
`;
